extensions/2.0/Vendor/EXT_feature_metadata/schema/primitive.EXT_feature_metadata.featureMapping.featureIds.schema.json

```diff
@@ -0,0 +1,39 @@
+{
+    "$schema": "http://json-schema.org/draft-04/schema",
+    "title": "Per-vertex feature IDs",
+    "type": "object",
+    "description": "An object describing per-vertex feature IDs to be used as indices to property arrays in the feature collection.",
+    "allOf": [ { "$ref": "glTFProperty.schema.json" } ],
+    "properties": {
+        "attribute": {
+            "type": "string",
+            "pattern": "^_FEATURE_ID_([1-9]\\d*|0)$",
+            "description": "The name of the attribute containing feature IDs. Feature IDs must be whole numbers in the range `[0, elementCount - 1]`, where `elementCount` is the total number of elements in each property array."
+        },
+        "constant": {
```

Comment:
As the TODO here says, `constant` could be better named. Maybe... `firstFeatureId`?

Reply:
Somehow I want the naming to convey that the feature ID grabbed for a vertex is **this** value, in lieu of an accessor.

I think `firstFeatureId` has the built-in assumption that `vertexStride` will be used, which may not always be the case.

Reply:
We talked about this, still haven't reached a definite answer yet.

Some thoughts:
* locally I have a chess pieces model with many separate meshes, this would be a helpful example to determine how useful stride is
* setting stride to 0 would be useful in that case where you want the vertices of a single piece to all be the same.